extensions/podman/packages/extension/src/extension.ts

```
@@ -1777,6 +1785,30 @@ export async function calcPodmanMachineSetting(): Promise<void> {
+ * Related issue: https://github.com/podman-desktop/podman-desktop/issues/17488
+ */
+export async function getMemoryUsedFromMemAvailable(
+  machineName: string,
+  vmType: string,
+  memTotal: number,
+): Promise<number | undefined> {
+  try {
+    const { stdout } = await execPodman(['machine', 'ssh', machineName, 'cat /proc/meminfo'], vmType);
+    const match = /MemAvailable:\s+(\d+)\s+kB/.exec(stdout);
+    if (match) {
+      const memAvailableBytes = parseInt(match[1], 10) * 1024;
```

Biome (2.5.1)

[error] Use Number.parseInt instead of the equivalent global.

(lint/style/useNumberNamespace)